.mergify.yml

```diff
@@ -7,53 +7,14 @@ pull_request_rules:
       - "#review-requested=0"
       - "#changes-requested-reviews-by=0"
       - label!=status:block-merge
+      - label=status:merge-when-green
```

Comment:
So a simple `Approve` won't be enough for mergify to kick in? 

This means a maintainer will now have to (1) `Approve` and (2) add the label. I find it's too much work.

Instead, the current approach: "Hey, I like 99% of this so I'll (1) `Approve` but it needs XYZ so (2) I'll mark `block-merge` to give you time to address it". It's the same amount of clicks and it doesn't penalize the most common case "It looks 100% good to me".

PS: I agree with all the rest of the cleanup. I just doubt about this new condition.

Reply:
> "Hey, I like 99% of this so I'll (1) Approve but it needs XYZ so (2) I'll mark block-merge to give you time to address it"

Except that this is not the practice in the team (Akka team), we should have only one way of working and it's annoying to have to do it different in Play and Lagom.

We do two clicks in Akka and any other project under Akka org. One to approve, one to merge. With that change, you can: 

* approve and merge if the build is already green (2-clicks)
* approve and label if build is still running (2-clicks).

And there is still one more thing, if you approve and add the label `block-merge`, there is the risk that mergify beats you. So, you have to go back to "Converstaion" tab (1-click), add the label `block-merge` (1-click), go back to "Files changed" (1-click) and "Approve" (1-click). Total 4 clicks.

Reply:
💯 merging should be an explicit and informed decision (clicking the button or adding a label explicitly for that serves that purpose)